src/Maker/MakeEntity.php

```diff
@@ -881,16 +882,4 @@ private function getEntityNamespace(): string
     {
         return $this->doctrineHelper->getEntityNamespace();
     }
-
-    private function getTypesMap(): array
-    {
-        $types = Type::getTypesMap();
-
-        // remove deprecated json_array if it exists
-        if (\defined(sprintf('%s::JSON_ARRAY', Type::class))) {
-            unset($types[Type::JSON_ARRAY]);
-        }
```

Comment:
Are sure about this? Won't users still be potentially using a `doctrine/dbal` that includes the deprecated `JSON_ARRAY`?

Reply:
good catch, we'll deal with that removal in a separate PR